indigo_app/views/places.py

```diff
@@ -301,11 +301,14 @@ def get(self, request, *args, **kwargs):
         self.form.is_valid()
 
         if params.get('format') == 'xlsx':
-            return self.generate_xlsx()
-        
+            self.full_index = params.get('full-index')
```

Comment:
Once the button has moved into the settings view, the logic for the full export could move to a separate view completely. There's really no link between that export and this view/our existing xlsx export.